sql/src/main/java/io/crate/execution/dml/ShardResponse.java

```diff
@@ -91,6 +91,8 @@ public String toString() {
 
     private IntArrayList locations = new IntArrayList();
     private List<Failure> failures = new ArrayList<>();
+    private List<Object> values = new ArrayList<>();
```

Comment:
Maybe we could be a bit more verbose here. `values` is a bit generic and doesn't contain any information about the format and what kind of values. Are these the columns, or the rows?